pkg/util/procfs/procfs_test.go

```diff
@@ -56,3 +59,11 @@ func TestContainerNameFromProcCgroup(t *testing.T) {
 	procCgroupInvalid := "devices:docker/kubelet\ncpuacct:pkg/kubectl"
 	verifyContainerName(procCgroupInvalid, "", true, t)
 }
+
+func TestPidOf(t *testing.T) {
+	if runtime.GOOS == "darwin" || runtime.GOOS == "windows" {
+		t.Skipf("not supported on GOOS=%s", runtime.GOOS)
+	}
+	pids := PidOf("init")
+	assert.Equal(t, []int{1}, pids)
```

Comment:
Ack. pushing an update to see what exactly is present there in the integration environment